app/controllers/ride_mechanics_controller.rb

```diff
@@ -0,0 +1,11 @@
+class RideMechanicsController < ApplicationController
+  def create
+    RideMechanic.create!(ride_mechanic_params)
+    redirect_to mechanic_path(params[:ride_mechanic][:mechanic_id])
+  end
```

Comment:
Careful with adding `!` (bangs) into your code. They should only be used if you are using exception handlers. Otherwise, if for whatever reason they fail, it will render a really ugly error to our end user.